docs/CONTRIBUTING.md

```diff
@@ -14,17 +14,31 @@ If you are a *git ninja*, we feel happy and sad for you at the same time. Your s
 ## Reviews
 Reviews are a great way to get familiar with the codebase. If you would like to start with reviewing just use **[this handy github filter](https://github.com/xbmc/xbmc/pulls?q=is%3Apr+is%3Aopen+review%3Anone)** to see PRs that are in need of one (we have plenty!).
 
-## Pull request guidelines
+## Pull request submission guidelines
 * **Create topic branches**. Never ever open a pull request from your master branch. **Ever!**
 * **Code against master branch**. Unless release specific, all Kodi development occurs in the `master` branch. If it needs backporting it can be done after it hits master.
 * **One topic branch per change**. Found something that needs fixing but is unrelated to the current work? Create a new topic branch and open another *pull request*.
 * **Use a coherent commit history**. Make sure each individual commit in your *pull request* is meaningful and organized in logical chunks. Tidy up and squash commits before submitting.
-* **Be descriptive, but concise**. The *pull request* and commits should have a useful title and follow the `[component(s)] Short description...` format. No, `[README.md] Update` is **not** descriptive enough. The *pull request* description should only contain information relevant to the change.
-* **Be patient, be kind, rewind**. We're all volunteers with busy lives but usually very responsive in cataloging *pull requests* and pinging the relevant maintainer(s). Sometimes though, things go unnoticed for longer than usual. Probably due to beer deprivation... or lack thereof. Feel free to nudge us if we forget.
+* **Be descriptive, but concise**. The *pull request* and *commits* should have an useful title and follow the format:<br/>
```

Comment:
```suggestion
* **Be descriptive, but concise**. The *pull request* and *commits* should have a useful title and follow the format:<br/>
```
"a useful title" is correct.

Reply:
I was thinking "an" was used before each vowel i remember wrong

Reply:
Yes, that is the general rule, but it applies to the sound and not the writing. "Useful" is pronounced /ˈjuːs.fəl/ and starts with the consonant "j", which is why it is "a useful [something]" :-)

See also https://dictionary.cambridge.org/grammar/british-grammar/a-an-and-the